### PR TITLE
Add functionality to the Reservation screen

### LINKED ISSUE #16: Add functionality to the Reservation screen
```[tasklist]
### Tasks
- [ ] Users must be able to see a list that shows the information of their flight reservations.
- [ ] The reservation information must be retrieved from Firebase.
```

### PR DESCRIPTION
**Pull request by Axw3ll**
In this issue, we added functionality to the FlatList component to be able to view the flights stored in the database. We achieved this by querying Firebase and specifying that we wanted the data related to the current user's ID. We used the useEffect hook to ensure that the query is only executed once and not on every render.

# Description
-Introduces the GetFlights function which retrieves flights for the current user. It uses Firestore and Authentication from  Firebase. It imports necessary modules, defines GetFlights as an async function, fetches the current user, queries the 'flights' Firestore collection filtered by user's UID, retrieves the matching flights, and stores them in an array. It then logs and returns the flights array. Errors during execution are logged and rethrown.

## Screenshots
imagen

imagen

### CODE REVIEW

**Review by Eduardoespl — APPROVED** [review]
LGTM! 🚀

**Review by Lizzflores — APPROVED** [review]
LGTM!👍

File: src/components/Card.jsx [hunk 1]
```diff
@@ -1,29 +1,34 @@
...
-//5974f5
-const Card = () => {
+
+
+//date
+//destinationCity
+//destinationCountry
+//originCity
+//originCountry
+//passengers
```
Comment (xFerchoVJ) [inline src/components/Card.jsx]: Try not to leave comments on your PR

**Review by xFerchoVJ — COMMENTED** [review]
LGTM, just remove the comments :)

File: src/components/Card.jsx (hunk 1 above)
Comment (BrandonVG) [reply src/components/Card.jsx]: We deleted the comments

File: src/components/Card.jsx [hunk 2]
```diff
@@ -1,29 +1,27 @@
...
+                <Text style={styles.ciudad} >{destinationCity}</Text>
+                <Text style={styles.pais} >{destinationCountry}</Text> 
+                <Text style={styles.infoAdd}> {passengers} passengers</Text>
             </View>
         </View>
         <View style={styles.linea1}></View>
     </View>
-    
   )
 }
```
Comment (YilKanda) [inline src/components/Card.jsx]: Check the indentation of your file

File: src/components/Card.jsx [hunk 3]
```diff
@@ -33,11 +31,14 @@ const styles = StyleSheet.create({
...
         height:120,
         flexDirection:'row',
+        alignSelf:'center',
         justifyContent:'space-between',
         marginTop:8,
+        borderBottomColor:'gray',
+        borderBottomWidth:2
     },
     infOne:{
         width:'40%',
```
Comment (YilKanda) [inline src/components/Card.jsx]: The styles must be in a separate file.

File: src/components/FlightList.jsx [hunk 4]
```diff
@@ -1,17 +1,36 @@
...
-    )
+const FlightList = () => {
+    const [flights, setFlights] = useState([])
+    useEffect(() =>{
+        const getFlightsData = async () => {
+            const getFlights = await GetFlights();
+            setFlights(getFlights);
+        }
+        getFlightsData();
+    },[]);
```
Comment (YilKanda) [inline src/components/FlightList.jsx]: Check the indentation

File: src/hooks/GetFlights.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+    const flightsCollection = await firestore()
+    .collection('flights')
+    .where('userId', '==', currentUser.uid)
+    .get();
+    const flights = [];
+    flightsCollection.forEach(documentSnapshot => {
+      const flight = documentSnapshot.data();
+      flights.push(flight);
+    });
+    console.log(flights);
```
Comment (YilKanda) [inline src/hooks/GetFlights.js]: Don't leave debug code on GitHub, like console.log(), you can use Alert.alert() instead

File: src/hooks/GetFlights.js [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+    return flights;
+  }
+  catch(error){
+    console.log(error);
```
Comment (YilKanda) [inline src/hooks/GetFlights.js]: Don't leave debug code on GitHub, like console.log(), you can use Alert.alert() instead

File: src/screens/HomeScreen.jsx [hunk 7]
```diff
@@ -8,12 +8,13 @@ import handleLogOut from '../hooks/HandleLogOut';
...
 import AddButton from '../components/AddButton';
 
+
 const HomeScreen = () => {
   const user = OnAuthStateChanged();
-  const navigation = useNavigation();
+  const navigation = useNavigation();  
 
   const handleOnLogOut = async () => {
     console.log('handleOnLogOut executed');
```
Comment (YilKanda) [inline src/screens/HomeScreen.jsx]: Don't leave debug code on GitHub, like console.log(), you can use Alert.alert() instead

**Review by YilKanda — CHANGES_REQUESTED** [review]

**Pull request merged**